Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -64,11 +65,15 @@ class RubricCircleView: UIView {
             let font: UIFont
             let color: UIColor
             let bgColor: UIColor
+            let format = NSLocalizedString("g_points", bundle: .core, comment: "")
+            var a11yLabel: String = String.localizedStringWithFormat(format, Int(r))
+            a11yLabel += description
 
             if selected {
                 font = UIFont.scaledNamedFont(.semibold20)
                 color = UIColor.white
                 bgColor = Brand.shared.primary
+                a11yLabel += NSLocalizedString("Selected", bundle: .core, comment: "")
```

Comment:
Would `button.selected = true` give you the same or better result in VO?

Reply:
Oh good suggestion, I will give it a try.

Reply:
Yeah I like this better.